t/node/plugin-configs.t

```diff
@@ -123,3 +123,38 @@ qr/conf_version: \d+#\d/
 qr/conf_version: \d+#1
 conf_version: \d+#2
 /
+
+
+
+=== TEST 2: validated plugins configuration via incremental sync
```

Comment:
You may add some test cases that written some malformed data to etcd and detect the error log in APISIX.

Reply:
added. @tokers